utils/getEventId.js

```diff
@@ -0,0 +1,4 @@
+function getEventId(message){
+    return message.traits.event_id || message.context.traits.event_id || message.properties.event_id || message.messageId;
```

Comment:
Can you move this file to FacebookPixel folder as its specif to its use case. And rename this file as util.js

Reply:
Done